### ISSUE TITLE
Simplify workflow for updating documentation

### ISSUE THREAD

**Issue by brookslogan** [cmu-delphi/epidatasets#14]
Migrating from memory of meeting.

Updating just **documentation** for data objects currently requires:
- Updating docs in epidatasets.
- Re-documenting downstream packages with latest epidatasets, as downstream packages inherit documentation.
- Every developer pulling epidatasets to not reverse doc updates.
- (Maybe some other things?  Adjusting version bounds doesn't seem strictly necessary since it's just documentation, but might assist in forcing developers to stay in sync.)

Ideas for simplifying:
- Replace downstream docs with just a single data re-exports topic with links to epidatasets documentation for each object.  We get to keep the note about `data(<obj name>)` (and could add one about needing epidatasets attached for listing its data sets with just `data()`).
- Remove downstream docs; just `@export` and that's it.  `?`/`help` should route to epidatasets documentation [since we `Imports:` it], assuming we have identical object naming.  We lose the notes about not being able to use `data(<obj name>)`, but that's an dated paradigm anyway, so hopefully not a huge loss.
- Don't re-export at all downstream.  This requires adding `epidatasets::` and/or `library(epidatasets)` in various downstream examples and vignettes.

**Comment by brookslogan** [comment]
I'm trying out the only-`@export` approach in https://github.com/cmu-delphi/epiprocess/pull/564 because CHECK was complaining about `{epiprocess}` in the copied documentation.  (Trying to parse it as .Rmd stuff.)

**Comment by brookslogan** [comment]
It's a fight between roxygen2, CHECK, and pkgdown to get something that looks okay and passes all checks.  I've ended up with
- an internal re-export topic in epiprocess that I can't seem to get rid of the Format: section in (`@format NULL` doesn't appear to work)
- direct links to epidatasets in the pkgdown reference

**Comment by dsweber2** [comment]
seems pretty solved to me, especially with `epidatasets` fairly stable. Move to close?

**Comment by brookslogan** [comment]
Yeah, I think @nmdefries pushed a solution to this to both epiprocess & epipredict.  (I also drafted another approach in some miscellaneous merged epiprocess PR to take care of some CHECK/pkgdown stuff if we ever need to revisit the impl here.)
